x11-misc/redshift-qt/redshift-qt-0.4.ebuild

```diff
@@ -0,0 +1,38 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+DESCRIPTION="Redshift QT5 Based Frontend"
+HOMEPAGE="https://github.com/Chemrat/redshift-qt/"
+SRC_URI="https://github.com/Chemrat/${PN}/archive/v${PV}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="+icons"
+
+DEPEND="dev-qt/qtgui:5
+		dev-qt/qtcore:5
+		dev-qt/qtwidgets:5"
+RDEPEND="${DEPEND}
+		x11-misc/redshift
+		icons? ( x11-misc/redshift[-gtk] )"
```

Comment:
what's the deal with requiring `-gtk`? that will end up as a blocker for many users.

Reply:
if you want to use the icons in this package they will collide with the icons in the gtk UI

Reply:
couldn't say.. I suppose two options that I don't like

1.always require gtk and then use those icons
2.check if gtk use flag is used and if not then install icons